src/lib.rs

```diff
@@ -265,7 +271,9 @@ mod dl {
     use std::result::Result::{Ok, Err};
     use std::string::String;
     use std::vec::Vec;
-    use std::sys::c::compat::kernel32::SetThreadErrorMode;
+    use kernel32::SetThreadErrorMode;
+    use kernel32;
```

Comment:
It's kinda weird being inconsistent in `use`ing a thing directly in some places, and using a path to refer to it in other places.